styles/styles.css

```diff
@@ -406,6 +406,30 @@ main .section.padded > .section-container {
   margin-left: auto;
 }
 
+.equitypage h1, h2, p {
```

Comment:
Do we need `.equitypage` ? I think the global styles should look the same across pages. But even if they were page specific, we can add this later with page templating.

Reply:
done removed the page specific css